pkg/build/util/util_test.go

```diff
@@ -188,7 +189,7 @@ func checkEnvList(t *testing.T, envs s2iapi.EnvironmentList, orig bool) {
 				if credsRegex.MatchString(env.Value) {
 					t.Errorf("credentials not stripped from env value %v", env)
 				}
-				if !redactedRegex.MatchString(env.Value) {
+				if isURL(env.Value) && urlStringHasPassword(env.Value) && !redactedRegex.MatchString(env.Value) {
```

Comment:
isn't urlStringHasPassword always going to be false if the redaction worked, so the redactedRegex check will never happen?

Reply:
good catch. The redacted regex won't run in this case, but we are assured that the url string doesn't contain a password (may have a username)

Reply:
just removing the redactedRegex check is probably fine.  I dno't want to leave it there to potentially confuse someone in the future though.

or you can go for bonus points and fix the logic to actually invoke that check.